skyportal/handlers/api/source.py

```diff
@@ -114,6 +118,44 @@
 Session = scoped_session(sessionmaker())
 
 
+def run_async(func, *args, **kwargs):
```

Comment:
Can we put this in utils/ or models/ or somewhere where it might be useful beyond source.py?

Reply:
Yes, I was only trying to figure out where to put it! Will move it somewhere better

Reply:
done @mcoughlin, also fixed some stuff here and there, and made sure the position shown on the frontend updates when we modify it in the DB.